WS17_18/01_introduction/your assignment/hahn/Yatodoa/Yatodoa/ViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ViewController.swift
+//  Yatodoa
+//
+//  Created by Alex on 24.10.17.
+//  Copyright © 2017 Alexander Dobrynin. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    @IBOutlet weak var doneLabel: UILabel!
+    
+    var todos: [String] = [] {
+        didSet {
+            print(todos)
+            if(todos.count == 1) {
+                self.doneLabel.text = String(todos.count) + " Aufgabe erledigt"
```

Comment:
In Swift kann man auch String-Interpolation nutzen. Das würde das so aussehen: `self.doneLabel.text =  "\(todos.count) Aufgabe erledigt"`

Man kann sogar ganze Ausdrücke interpolieren: `self.doneLabel.text =  "\(todos.count) \(todos.count == 1 ? "Aufgabe" : "Aufgaben") erledigt`